crates/xline/src/server/lease_server.rs

```diff
@@ -333,7 +333,7 @@ where
             // a follower when it lost the election. Therefore we need to double check here.
             // We can directly invoke leader_keep_alive when a candidate becomes a leader.
             if !self.lease_storage.is_primary() {
-                let leader_addrs = self.cluster_info.addrs(leader_id).unwrap_or_else(|| {
+                let leader_addrs = self.cluster_info.client_urls(leader_id).unwrap_or_else(|| {
```

Comment:
Why this need to use client_urls?

Reply:
Because here we need to forward the request to the LeaseServer,